internal/server/data/migrator/helpers.go

```diff
@@ -15,7 +15,7 @@ func HasTable(tx DB, name string) bool {
 		SELECT count(*)
 		FROM information_schema.tables
 		WHERE table_schema = CURRENT_SCHEMA()
-		AND table_name = ? AND table_type = 'BASE TABLE'
+		AND table_name = $1 AND table_type = 'BASE TABLE'
```

Comment:
This commit removes gorm entirely from the `internal/server/data/migrator` package. With gorm removed we have to use the PostgreSQL placeholder in queries which is `$n` instead of `?`.

We'll have to make this change in the `data` package as well, but that's going to be a bigger effort. It was easy enough to change the few queries in this package now.